scan/debian.go

```diff
@@ -240,18 +240,25 @@ func (o *debian) rebootRequired() (bool, error) {
 
 func (o *debian) scanInstalledPackages() (models.Packages, models.Packages, error) {
 	installed, updatable := models.Packages{}, models.Packages{}
-	r := o.exec("dpkg-query -W", noSudo)
+	r := o.exec("dpkg-query -W -f=\"\\${binary:Package}\\t\\${db:Status-Abbrev}\\t\\${Version}\n\"", noSudo)
```

Comment:
```
 r := o.exec(`dpkg-query -W -f='${binary:Package}\t${db:Status-Abbrev}\t${Version}\n\'`, noSudo)
```
It is not necessary to escape the Go language string by enclosing with ``. Also, using '' does not have to escape on the shell.

Reply:
Good to know. Let me try it out.